primitives/ethereum/Cargo.toml

```diff
@@ -35,6 +35,7 @@ ethereum-types = { version = "0.12", features = ["codec"] }
 keccak-hasher  = { version = "0.15" }
```

Comment:
After this pr,  we can delete this crate IMO. Only used  as `use ethereum_primitives::{H160, H256, U256};`. There are many replacement solutions.

Reply:
#1456

Reply:
Why not do it in this PR together?

Reply:
I prefer to open another PR to do the cleaning job.